File: fixtures/c1/language/typescript/declarations.ts
// UTF-8 注释用于冻结字节与行列位置：你好 😀
export class Formatter<T> {
  format(value: number): number { return value; }
  format(value: string): string { return value; }
  echo<TValue>(value: TValue): TValue { return value; }
}

export class Secondary {
  format(value: number): number { return value; }
}

export function format(value: string): string { return value; }

File: fixtures/c1/language/csharp/declarations.cs
namespace Demo.Unicode;

// UTF-8 注释用于冻结字节与行列位置：你好 😀
public class 格式化器<T>
{
    public int Format(int value) => value;
    public string Format(string value) => value;
    public T Echo<TValue>(TValue value) => default!;
}

public class Secondary
{
    public int Format(int value) => value;
}
